composer.json

```diff
@@ -13,10 +13,10 @@
         "forum": "https://discourse.laminas.dev/"
     },
     "require": {
-        "php": "^7.3 || ^8.0"
+        "php": "^7.4 || ~8.0.0 || ~8.1.0"
     },
     "require-dev": {
-        "laminas/laminas-coding-standard": "^2.2",
+        "laminas/laminas-coding-standard": "~2.3.0",
         "phpunit/phpunit": "^9.5"
```

Comment:
Can we fix `phpunit.xml` to have the latest `9.5` series?

 * `<whitelist/>` element no longer exists ( https://github.com/mezzio/repo-template/blob/d1c44b7cb18686f11884db211729d79d47f99e68/phpunit.xml.dist#L12-L16 )
 * we need to configure it to detect deprecations (attribute to be added in https://github.com/mezzio/repo-template/blob/d1c44b7cb18686f11884db211729d79d47f99e68/phpunit.xml.dist#L2-L5 )

Reply:
Sure, but can I in separate PR?

Reply:
OK, merging here then, thanks!